packages/expo-barcode-scanner/src/BarCodeScanner.tsx

```diff
@@ -62,9 +62,15 @@ export type BarCodeScannerResult = {
    */
   type: string;
   /**
-   * The information encoded in the bar code.
+   * The parsed information encoded in the bar code.
    */
   data: string;
+  /**
+   * The raw information encoded in the bar code.
+   * May be different from `data` depending on the barcode type.
+   * Will always be undefined on iOS.
```

Comment:
```suggestion
   @platform android
```

it's quite advanced feature, i would like to add `@hidden` to hide this from the [doc](https://docs.expo.dev/versions/latest/sdk/bar-code-scanner/). how do people think about this?

Reply:
@Kudo Is the `@hidden` flag meant to completely hide the field reference and documentation?

Reply:
it should expose to typescript with autocomplete and show the property comment if mouseover the property. just not into the doc.